mysql-test/suite/innodb/r/innodb-alter-timestamp.result

```diff
@@ -6,17 +6,14 @@ ERROR 01000: Data truncated for column 'i1' at row 1
 ALTER TABLE t1 CHANGE i1 i1 INT UNSIGNED NOT NULL DEFAULT rand(),
 ALGORITHM=COPY;
 ERROR 01000: Data truncated for column 'i1' at row 1
-ALTER TABLE t1 CHANGE i1 id INT UNSIGNED NOT NULL AUTO_INCREMENT,
-ADD PRIMARY KEY(id), ALGORITHM=INPLACE;
-ERROR 0A000: ALGORITHM=INPLACE is not supported. Reason: Cannot change column type INPLACE. Try ALGORITHM=COPY
```

Comment:
I would keep the no-longer-failing statement and add another one that removes the `AUTO_INCREMENT` attribute, possibly with `ALGORITHM=INSTANT`.
Yes, you did add a separate test `innodb.instant_auto_inc` that covers this functionality, but I would prefer not to delete or modify statements in existing tests unless there is a compelling reason.